cms/djangoapps/contentstore/views/item.py

```diff
@@ -98,6 +98,7 @@ def xblock_handler(request, usage_key_string):
     GET
         json: returns representation of the xblock (locator id, data, and metadata).
               if ?fields=graderType, it returns the graderType for the unit instead of the above.
+              if ?course_tree=1, it returns the a course_outline summary and XBlock summary.
```

Comment:
I don't like 0/1 for boolean parameters. Our standard convention is ``true``/``false``, such as here:

https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/views.py#L2038

Interestingly, this isn't documented in our current API best practices, so I added a comment about it on @efagin's OEP: https://github.com/edx/open-edx-proposals/pull/36